terraform/environments/mlra/cloudwatch.tf

```diff
@@ -0,0 +1,28 @@
+module "cwalarm_1" {
+  source = "./modules/cloudwatch-alarms"
+
+  alarmname  = "elb-4xx-error-alarm"
+  oper       = "GreaterThanThreshold"
+  eval       = "5"
+  metricname = "HTTPCode_ELB_4XX_Count"
+  namespace  = "AWS/ApplicationELB"
+  period     = "60"
+  stat       = "Sum"
+  thresh     = "10"
+  alarmdesc  = "This alarm will trigger if we receive 4 4XX elb alerts in a 5 minute period."
+
+}
+module "cwalarm_2" {
+  source = "./modules/cloudwatch-alarms"
+
+  alarmname  = "elb-4xx-error-alarm"
+  oper       = "GreaterThanThreshold"
+  eval       = "5"
+  metricname = "HTTPCode_ELB_4XX_Count"
+  namespace  = "AWS/ApplicationELB"
+  period     = "60"
+  stat       = "Sum"
+  thresh     = "10"
+  alarmdesc  = "This alarm will trigger if we receive 4 4XX elb alerts in a 5 minute period."
+
+}
```

Comment:
Is this file required? We would ideally be calling the monitoring module from main.tf. With that, this arrangement would call the module twice, but which resources in the module does this feed into?

Reply:
file has been modified for now but called module can be moved to main when ready